splash/defaults.py

```diff
@@ -9,7 +9,7 @@
 
 # Default size of browser window.  As there're no decorations, this affects
 # both "window.inner*" and "window.outer*" values.
-VIEWPORT_SIZE = '1024x768'
+VIEWPORT_SIZE = '1920x1080'
```

Comment:
The advantage of a smaller value is that you'll get a smaller screenshot by default. Also, 1920x1080 is not the most popular size - 1366x768 looks more popular. What do you think about changing it to 1366x768? It should be done in another PR: this is a backwards-incompatible change, and it'd be good to hear other's opinions.